exercises-java/exercise1/README.md

```diff
@@ -46,7 +46,7 @@ Before you can start using SAP Application Studio, you need to create your devel
 
     ```bash
     mvn -B archetype:generate -DarchetypeArtifactId=cds-services-archetype -DarchetypeGroupId=com.sap.cds \
-    -DarchetypeVersion=1.0.1 -DcdsVersion=3.17.4 \
+    -DarchetypeVersion=1.2.0 -DcdsVersion=3.21.2 \
     -DgroupId=com.sap.teched.cap -DartifactId=products-service -Dpackage=com.sap.teched.cap.productsservice
```

Comment:
We should replace this with a shorter call to `cds init`.  I guess, you prefer to still have the fixed archetype and cds versions in it, right?

Reply:
Yes, but the other problem is that we want to use version 1.2.0, but the CDS-DK version available on AppStudio only provides 1.1.0. Also we don't want to initialize sample data, which was surprisingly added by cds init by default.

Reply:
So this means if we add the 3 parameters `archetypeVersion`, `cdsVersion`, and `samples` (or so) to cds init, it would work?
Would be something like
```
cds init products-service --add java --java:archetypeVersion=1.2.0 --java:cdsVersion=3.21.2
```
`samples` should default to `false` anyways, to align it with Node.js

Reply:
Yes, that would be great :)

Reply:
Ok, I will align this with Joerg once I know his github user.

For the time being, we can leave the archetype call here.